package.json

```diff
@@ -52,8 +51,12 @@
     "grunt-sass": "latest",
     "grunt-unused": "latest",
     "grunt-usemin": "latest",
-    "jit-grunt": "^0.10.0",
-    "phplint": "^1.7.4",
     "push.js": "^1.0.4"
+  },
+  "devDependencies": {
```

Comment:
I haven't done an in depth code review yet, just a quick question.
Why did you move these dependencies to devDependencies?

Will probably be my small npm knowledge that is failing here.

Reply:
That was on accident though we shjould definitely look through these. dependencies are required to run the application where as develop dependencies are only needed to develop. Things like phplint and other things that dont need to be in the build process shouldn't be in there. I'll move them back for now and we can discuss at a later time.